outputs.tf

```diff
@@ -1,6 +1,19 @@
-output "private_endpoints" {
-  description = <<DESCRIPTION
-  A map of the private endpoints created.
-  DESCRIPTION
-  value       = var.private_endpoints_manage_dns_zone_group ? azurerm_private_endpoint.this_managed_dns_zone_groups : azurerm_private_endpoint.this_unmanaged_dns_zone_groups
+output "name" {
+  description = "The name of the Service group."
+  value       = azapi_resource.service_group.name
+}
+
+output "resource" {
+  description = "This is the full output for the Service group."
+  value       = azapi_resource.service_group
```

Comment:
please don't output the whole resource, these can change and break downstream consumers. Be specific about the things you want